src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -122,6 +123,10 @@ protected function configure(): void
 Use the --all option to consume from all receivers:
 
     <info>php %command.full_name% --all</info>
+
+Use the --exclude-receivers option to exclude specific receivers/transports from consumption (can only be used with --all):
```

Comment:
```suggestion
Use the <info>--exclude-receivers</info> option to exclude specific receivers/transports from consumption (can only be used with <info>--all</info>):
```

Reply:
Why this change ? the other comment parameter not apply the info tag in the use description only on the command exemple.

```
Use the --bus option to specify the message bus to dispatch received messages
to instead of trying to determine it automatically. This is required if the
messages didn't originate from Messenger:

    <info>php %command.full_name% <receiver-name> --bus=event_bus</info>

Use the --queues option to limit a receiver to only certain queues (only supported by some receivers):

    <info>php %command.full_name% <receiver-name> --queues=fasttrack</info>
```

Reply:
Or I suggest to update all the file with our suggest (add a tag info in every parameter use exemple)

Reply:
Based on
* https://github.com/symfony/symfony/pull/61446